content/chapter-03/ApplicationMetrics.md

```diff
@@ -9,13 +9,13 @@
   * Availability percentage, e.g.: Percentage of uptime for interval of time
```

Comment:
Availability: percentage of uptime per time interval

Reply:
Done